pyrasterframes/src/main/python/docs/reference.pymd

```diff
@@ -65,6 +65,15 @@ See also GeoMesa [st_envelope](https://www.geomesa.org/documentation/user/spark/
 
 Convert an extent to a Geometry. The extent likely comes from @ref:[`st_extent`](reference.md#st-extent) or @ref:[`rf_extent`](reference.md#rf-extent).
 
+
+### rf_spatial_index
+
+    Long rf_spatial_index(Geometry geom, CRS crs)
+    Long rf_spatial_index(Extent extent, CRS crs)
+    Long rf_spatial_index(ProjectedRasterTile proj_raster, CRS crs)
+    
+Constructs a XZ2 index in WGS84/EPSG:4326 from either a Geometry, Extent, ProjectedRasterTile and its CRS.
```

Comment:
Perhaps add a bit about its intent / use.

```suggestion
Constructs a XZ2 index in WGS84/EPSG:4326 from either a Geometry, Extent, ProjectedRasterTile and its CRS. This column expression may be useful for [range partitioning](https://www.waitingforcode.com/apache-spark-sql/range-partitioning-apache-spark-sql/read).
```

Reply:
Can also point to the official docs if we want: http://spark.apache.org/docs/latest/api/python/pyspark.sql.html?highlight=registerjava#pyspark.sql.DataFrame.repartitionByRange